libsolidity/ast/Types.cpp

```diff
@@ -1478,11 +1478,6 @@ TypePointer StringLiteralType::mobileType() const
 	return TypeProvider::stringMemory();
 }
 
-bool StringLiteralType::isValidUTF8() const
```

Comment:
Removed this helper because the file already uses `util::validateUTF8` in other places and seems like it provides little benefit, but more confusion when someone goes to edit the file.